frontend/src/App.js

```diff
@@ -1,15 +1,30 @@
-
-import './App.css';
+import "./App.css";
+import { useEffect } from "react";
 
 function App() {
+  var websocket = null;
+
+  useEffect(() => {
+    websocket = new WebSocket("ws://localhost:8000/ws");
+
+    websocket.onopen = () => websocket.send("Websocket Connected to React");
+
+    websocket.onmessage = function(event) {
+      console.log("Data received from backend: ", event.data);
+    };
+
```

Comment:
You want to send a message `onClose` as, let the client know if it needs to reload or just close connection.
```
    ws.onclose = (event) => {
      console.log("WebSocket connection closed: ", event);
    };
```